internal/pkg/policy/policy_output.go

```diff
@@ -229,6 +233,35 @@ func (p *Output) prepareElasticsearch(
 		output.PermissionsHash = p.Role.Sha2 // for the sake of consistency
 	}
 
+	if outputMap.GetMap(p.Name).GetString("type") == OutputTypeRemoteElasticsearch {
+		om, _ := outputMap.GetMap(p.Name).Marshal()
+		var outputObj map[string]any
+		json.Unmarshal(om, &outputObj)
```

Comment:
[Blocker]
was it a test or debug code? It seems `om` isn't used.

Reply:
`om` is being unmarshalled to `outputObj`
It might be possible to simplify this if I change the type of `outputMap` to contain `hosts` field as `[]string`

Reply:
I see, ok, but there is still the error being ignored. It needs to be checked, otherwise `outputObj` might not be suitable for use